library/src/main/java/com/novoda/downloadmanager/lib/DownloadsRepository.java

```diff
@@ -195,6 +198,71 @@ public void setDownloadSubmitted(FileDownloadInfo info) {
         contentResolver.update(info.getAllDownloadsUri(), contentValues, null, null);
     }
 
+    @Nullable
+    String getCurrentDownloadingBatchId() {
+        return getCurrentBatchIdWithColumnStatus(DownloadStatus.RUNNING);
```

Comment:
>we only support sequential downloads for now.

what do you mean? the library supports multiple downloads at a time at the moment, the value can be modified by specifying a value as metadata in the manifest `com.novoda.downloadmanager.MaxConcurrentDownloads`